lifecycle_test.go

```diff
@@ -300,6 +300,39 @@ func TestPreCreateModifierHook(t *testing.T) {
 			"Networking config's network ID should be retrieved from Docker",
 		)
 	})
+	
+	t.Run("Request contains exposed port modifiers", func(t *testing.T) {
+		// reqWithModifiers {
```

Comment:
Let's remove this comment, which is used by the docs page to render a code snippet by name
```suggestion
```

Reply:
@xmh19936688 I think when we remove this comments, we are ready to go 🚀

Reply:
Sorry for didn't see it before. Have done this yet.